taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/outbox/rest/controller/CamundaTaskEventsController.java

```diff
@@ -95,6 +95,18 @@ public Response decreaseRemainingRetriesAndLogError(
     return Response.status(204).build();
   }
 
+  @Path(Mapping.URL_UNLOCK_EVENT)
+  @POST
+  @Consumes(MediaType.APPLICATION_JSON)
+  public Response unlockEvents(
```

Comment:
this always unlocks exactely one event, right? If yes, what do you think about naming it unlockEvent?

Reply:
Yes, I will rename it